README.md

```diff
@@ -19,6 +19,7 @@ Once the draining process is complete, the instance will be terminated.
 #### Changable Parameters:
 * SCALE_IN_CPU_TH = 30 `# Below this EC2 average metric scaling would take action`
 * SCALE_IN_MEM_TH = 60 `# Below this cluster average metric scaling would take action`
+* FUTURE_CPU_TH = 40 `# Below this future metric scaling would take action`
```

Comment:
Let me know if you want to default this to 30 so that it doesn't break any existing functionality.

Reply:
@cbankston WDYT?
Plus, can you handle the conflicts?